### ISSUE TITLE
Claymore 9.7 zip download detected as dangerous and blocked

### ISSUE THREAD

**Issue by citirex** [nanopool/Claymore-Dual-Miner#54]
Hi,

I tried to download Claymore.s.Dual.Ethereum.Decred_Siacoin_Lbry_Pascal.AMD.NVIDIA.GPU.Miner.v9.7.zip several times both on chrome and firefox and this file is detected as dangerous by chrome. On firefox I get a message This file contains a virus or malware.
Is everything in order?

**Comment by dlethe** [comment]
Ignore the warning, assuming you are downloading from this official site

**Comment by citirex** [comment]
Hi, ignore is not an option only to discard the download. How can I
download it?

**Comment by dlethe** [comment]
Do the file SAVE instead of the OPEN function.  Then extract it by right-clicking file manager and running as administrator (the right-click is just in case, i do that as a general rule.  Also you might not be unable to zip it unless you have pkzip installed so heads up on that)

**Comment by citirex** [comment]
Yes Save option brings up an error. Any type of download unfortunately. Why
should this be happening?

**Comment by dlethe** [comment]
Turn off your virus detection temporarily. Wy does this happen?  Because whatever virus software you have was programmed to think that this code isn't safe.  

You should turn off all those services any and include defragmentation and indexing and so on for your mining machine. They just waste resources anyway.   (Unless you are using the machine for mining on weekends and evening LSI)

**Comment by MattPark** [comment]
Press control+J and it will give you the option to keep the file

**Comment by c2h2** [comment]
why is this happening? is a trojan or something infected?

**Comment by Reubenwelsh** [comment]
When the creator compiled the app he chose to do it in a way viruses and troyans usually are compiled to stop people from rebuilding it without the devfee - Many security experts have looked over this and deemed it safe. That being said,  it doesnt stop him from releasing a bad version in the future so always be careful when upgrading, wait a few days to make sure people have had time to have a look at it

**Comment by c2h2** [comment]
Say if I would like to make my own internal software for my company, run on 100+ PCs, Can I somehow wrap the exe and without letting anti-virus software saying anything? I am perfectly happy with paying dev fees.

**Comment by Reubenwelsh** [comment]
Not a globaly supported function (if it was possible, dont you think he would have done this with this package).

If this is your company though, there should be any issues deploying an exception to the computers to let this through, there is no need to turn the antivirus off, you can just tell it to trust this file or deploy it via a central deployment solution that is trusted.

**Comment by kebman** [comment]
My Chrome also attempted to stop the download. Right clicking and Save As did not help, though making a copy of the temp file worked. Will try ctrl+j in the future. I checked Claymore.s.Dual.Ethereum.Decred_Siacoin_Lbry_Pascal.AMD.NVIDIA.GPU.Miner.v9.8.zip and Panda Anti-Virus found three (3) trojans in it. The infected files were (allegedly) EthDcrMiner64.exe, Cuda7.5/EthDcrMiner64.exe and Cuda6-5/EthDcrMiner64.exe. Gonna look into the forums to see if it's still worth the risk.

**Comment by sivanov-hill30** [comment]
Here's the official download section in case someone would look for it. It wasn't obvious for me so may be some1 else is confused as well
https://github.com/nanopool/Claymore-Dual-Miner/releases
